kernel: bump 5.10 to 5.10.75

rsalvaterra opened the pull request:
Deleted (upstreamed):
bcm27xx/patches-5.10/950-0735-xhci-guard-accesses-to-ep_state-in-xhci_endpoint_res.patch [1]

Run-tested:
ramips/mt7621 (Redmi AC2100)

[1] https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=linux-5.10.y&id=dc3e0a20dbb9dbaa22f4a33dea34230f8c663c40

Signed-off-by: Rui Salvaterra \<rsalvaterra@gmail.com\>

rsalvaterra committed:
kernel: bump 5.10 to 5.10.73

Patches automatically refreshed.

Signed-off-by: Rui Salvaterra <rsalvaterra@gmail.com>

graysky2 commented:
```
    Build system: x86_64
    Build-tested: bcm2711/RPi4B, ipq806x/R7800
    Run-tested: bcm2711/RPi4B, ipq806x/R7800```

rsalvaterra committed:
kernel: bump 5.10 to 5.10.74

Patches automatically refreshed.

Signed-off-by: Rui Salvaterra <rsalvaterra@gmail.com>

graysky2 commented:
```
    Build system: x86_64
    Build-tested: bcm2711/RPi4B
    Run-tested: bcm2711/RPi4B
```

rsalvaterra committed:
kernel: bump 5.10 to 5.10.75

Deleted (upstreamed):
bcm27xx/patches-5.10/950-0735-xhci-guard-accesses-to-ep_state-in-xhci_endpoint_res.patch [1]

[1] https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=linux-5.10.y&id=dc3e0a20dbb9dbaa22f4a33dea34230f8c663c40

Signed-off-by: Rui Salvaterra <rsalvaterra@gmail.com>

graysky2 commented:
```
    Build system: x86_64
    Build-tested: bcm2711/RPi4B
    Run-tested: bcm2711/RPi4B
```
